### DISCUSSION TITLE
哪位大佬可以打包个Google calendar 吗？

### DISCUSSION THREAD

**Discussion by hubandad** [tw93/Pake#371] [Ideas]
每次打开chrome，一键关闭浏览器，总忘记还打开着Google calendar这茬。哎，有个app就好了。

**Comment by Tlntin** [comment]
自己动手，丰衣足食。[wiki](https://github.com/tw93/Pake/wiki/%E5%9C%A8%E7%BA%BF%E7%BC%96%E8%AF%91%EF%BC%88%E6%99%AE%E9%80%9A%E7%94%A8%E6%88%B7%E4%BD%BF%E7%94%A8%EF%BC%89)
